src/engraving/dom/textbase.cpp

```diff
@@ -1840,7 +1840,7 @@ void TextBase::createBlocks(LayoutData* ldata) const
                 state = 1;
                 token.clear();
             } else if (c == '&') {
-                state = 2;
+                state = 2; // why?
```

Comment:
Was this intended to be committed? 
(I think state 2 means entity parsing mode, i.e. `&amp;`, right?)

Reply:
Whoops, nope - that was a note to self 😅